test/plugins/test_yapf_format.py

```diff
@@ -68,3 +68,30 @@ def test_line_endings(workspace, newline):
     res = pylsp_format_document(doc)
 
     assert res[0]['newText'] == f'import os{newline}import sys{2 * newline}dict(a=1){newline}'
+
+FOUR_SPACE_DOC = """def hello():
+    pass
+"""
```

Comment:
Please move this variable to be after the import's section in this file because it's a constant.